scripts/git/main_to_google.sh

```diff
@@ -0,0 +1,17 @@
+set -x
+set -e
+set -o pipefail
+
+GREEN_COMMIT="${1:-main}"
+BRANCH="${BRANCH:-main-to-google}"
+
+git sync
```

Comment:
Suggested replacement for `git sync` custom alias:

```suggestion
git status --porcelain  # or git status --porcelain --ignore-submodules
git checkout main && git pull
```